modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
+User=znc
+Group=znc
+Restart=always
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=znc
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
OK, I'll test if it does what I think it does and update the docs if it really is problematic.